### ISSUE TITLE
How to put the generated pseudo mask into the segmentation model for training

### ISSUE THREAD

**Issue by CZeng1220** [linyq2117/CLIP-ES#32]
To train DeepLab-v2, we refer to [deeplab-pytorch](https://github.com/kazuto1011/deeplab-pytorch). The ImageNet pre-trained model can be found in [AdvCAM](https://github.com/jbeomlee93/AdvCAM).

Is there a more detailed process for step 3?
I would greatly appreciate any step-by-step guidance or example scripts. Thank you for your support!

**Comment by linyq2117** [comment]
Thanks for your interest!

To train a segmentation model using the generated masks, you need to change the data path. For example, the default data path for generated masks on VOC is `./output/voc12/pseudo_masks`.  To train the DeepLab-v2 based on [deeplab-pytorch](https://github.com/kazuto1011/deeplab-pytorch), you need to change the `data_root` in [configs/voc12.yaml](https://github.com/kazuto1011/deeplab-pytorch/blob/4219467fa5de07985f834f1bd8c04c186dc8f6d8/configs/voc12.yaml#L7) and `label_dir` in [libs/datasets/voc.py](https://github.com/kazuto1011/deeplab-pytorch/blob/4219467fa5de07985f834f1bd8c04c186dc8f6d8/libs/datasets/voc.py#L33). Then you can follow the `Training & Evaluation` scripts introduced in their `README.md`

**Comment by linyq2117** [comment]
Closed for inactivity.
